### PR TITLE
docs: upgrade architecture picture

### PR DESCRIPTION
**Pull request by fengjiachun**
## What's Changed in this PR

As the title said

## Checklist

- [ ] Please ensure that the content in `summary.yml` matches the current document structure when you changed the document structure.
- [ ] This change requires follow-up update in localized docs.

### CODE REVIEW

**Review by zhaoyingnan01 — DISMISSED** [review]
底色颜色在docs里有点过于深沉了，跟我们现在的扁平轻量化风格有些出入，可以把颜色换乘下面两种形式的。白底黑边黑字，或者浅色彩底黑字，辛苦稍修一下颜色
截屏2024-04-18 10 28 22
截屏2024-04-18 10 28 14

**Comment by fengjiachun** [comment]
PTAL @tisonkun @zhaoyingnan01 @nicecui

**Review by zhaoyingnan01 — APPROVED** [review]

File: docs/v0.7/en/contributor-guide/overview.md [hunk 1]
```diff
@@ -9,7 +9,7 @@
 - `Datanode` is responsible for storing data to persistent storage such as local disk or object storage in the cloud such as AWS S3, Azure Blob Storage etc.
 - `Metasrv` server that coordinates the operations between the `Frontend` and `Datanode`.
 
-![Architecture](/architecture.png)
```
Comment (nicecui) [inline docs/v0.7/en/contributor-guide/overview.md:12]: Can we just remove the `architecture.png` and the `artecture-2.png`, and `use architecture-3.png` as the new `architecture.png`?

Comment (fengjiachun) [reply docs/v0.7/en/contributor-guide/overview.md:12]: The documents before version 0.7 still need `architecture.png` and `architecture-2.png`.

**Review by nicecui — APPROVED** [review]

**Pull request merged**